pennylane/qchem/number.py

```diff
@@ -60,11 +61,20 @@ def particle_number(orbitals):
     r = np.arange(orbitals)
     table = np.vstack([r, r, np.ones([orbitals])]).T
 
-    coeffs = np.array([])
-    ops = []
+    sentence = FermiSentence({})
 
     for i in table:
```

Comment:
what is i? maybe a more descriptive name

Reply:
The code is modified (to remove traces of the old version). This function basically implements the number operator N = \sum_i a_i^\dagger a_i, which is a simple Fermi sentence.